### PR TITLE
chore: update cmake-utils

### PR DESCRIPTION
**Pull request by sfoster1**
Pull recent changes in cmake-utils into the subtree here and apply the new formatter changes.

### CODE REVIEW

File: stm32-modules/heater-shaker/include/heater-shaker/motor_task.hpp [hunk 1]
```diff
@@ -38,15 +38,13 @@ namespace motor_task {
...
-    ->std::same_as<int16_t>;
-    { cp.get_target_rpm() }
-    ->std::same_as<int16_t>;
+    { p.set_rpm(static_cast<int16_t>(16)) } -> std::same_as<errors::ErrorCode>;
+    { cp.get_current_rpm() } -> std::same_as<int16_t>;
+    { cp.get_target_rpm() } -> std::same_as<int16_t>;
     {p.stop()};
-    { p.set_ramp_rate(static_cast<int32_t>(8)) }
-    ->std::same_as<errors::ErrorCode>;
+    {
```
Comment (pmoegenburg) [inline stm32-modules/heater-shaker/include/heater-shaker/motor_task.hpp:45]: make 2 lines?

Comment (pmoegenburg) [reply stm32-modules/heater-shaker/include/heater-shaker/motor_task.hpp:45]: lines 45-47

**Review by pmoegenburg — APPROVED** [review]
looks a lot nicer

File: stm32-modules/heater-shaker/include/heater-shaker/motor_task.hpp (hunk 1 above)
Comment (sfoster1) [reply stm32-modules/heater-shaker/include/heater-shaker/motor_task.hpp:45]: that's the autoformatter doing that. if we've got some autoformatter config changes we should do them, but we shouldn't change back individual choices it makes.

**Pull request merged**